x-pack/solutions/security/plugins/security_solution/public/detections/constants.ts

```diff
@@ -13,3 +16,9 @@ export const SECURITY_ALERT_DATA_VIEW = {
 // Runtime field to extract the related_integration.package value from the kibana.alert.rule.parameters field
 // This is used in the EASE alert summary page
 export const RELATED_INTEGRATION = 'relatedIntegration';
+
+export const DETECTIONS_TABLE_IDS: TableIdLiteral[] = [
```

Comment:
Thank you for putting these here! It makes sense and it simplifies the few files where they're used!